src/Public/OpenApi.ps1

```diff
@@ -2104,7 +2104,11 @@ function New-PodeOABoolProperty {
         }
 
         if ($Default) {
-            $param.default = $Default
+            if ([bool]::TryParse($Default, [ref]$null) -or $Enum -icontains $Default) {
+                $param.default = $Default
+            } else {
+                throw "The default value is not a boolean and it's not part of the enum"
+            }
         }
```

Comment:
If `$Default` is an invalid boolean value, I'd just throw an error. You'll also want to test that the `$Enum` values are valid booleans as well.

Reply:
based on openapi spec enum can be anything.
for OpenApi and enum pizza,pasta,gelato,sushi are valid enum for boolean.
I changed the code here for exactly this reason

Reply:
`[bool]'pizza' = $true` sounds good to me 🍕!

Sounds crazy, but I guess if that's what the spec allows then fair enough!